packages/story-editor/src/components/panels/design/textStyle/stylePresets/stylePresets.js

```diff
@@ -156,6 +159,8 @@ function PresetPanel({ pushUpdate }) {
             <Icons.ChevronDownSmall />
           </StyledMoreButton>
           <Popup
+            topOffset={topOffset}
+            isRTL={isRTL}
```

Comment:
This is now getting cut off. 

<img width="605" alt="Screen Shot 2022-01-28 at 1 48 21 PM" src="https://user-images.githubusercontent.com/1820266/151619370-69e5b070-a731-4984-9cc5-ed8f1d07d24d.png">